playwright.config.js

```diff
@@ -50,11 +50,14 @@ export default defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
 
+  /* Limit the number of failures */
+  maxFailures: 1,
+
   /* Retry on CI only */
   retries: process.env.CI ? 1 : 0,
 
-  /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  /* Use a custom percentage of available wokers in CI and use default locally. */
+  workers: process.env.CI ? "75%" : undefined,
```

Comment:
That will be `3/4` workers in GitHub CI. Cranking this value up to `100%` led to intermittent application crashes.